app/adapters/application.js

```diff
@@ -0,0 +1,5 @@
+import DS from 'ember-data';
+
+export default DS.RESTAdapter.extend({
+  namespace: 'server/mocks'
```

Comment:
Namespace should be "api": see the route that is being registered with express for the HTTP Mock server